packages/devextreme/js/localization/messages/ar.json

```diff
@@ -195,6 +196,7 @@
         "dxDataGrid-masterDetail": "Cell with details",
 
         "dxTreeList-ariaTreeList": "Tree list with {0} rows and {1} columns",
+        "dxTreeList-ariaExpandableInstruction": "Use Ctrl + → key combination to expand and Ctrl + ← key combination to collapse a focused node",
```

Comment:
Question
Do all screen readers read non-ascii `←` char correctly?

Reply:
Yes, it is being pronounced "right/left arrow"

Reply:
Sorry, not sure. I only tried NVDA